bazel/toolchain/Dockerfile.sysroot

```diff
@@ -14,7 +14,8 @@ RUN apt-get update && apt-get install -y \
 
 # Copy required glibc files
 RUN mkdir -p /output/pkgs/{usr/,}lib/$(uname -m)-linux-gnu /output/pkgs/usr/include /output/pkgs/lib64 \
-    && cp -r /usr/lib/*-linux-gnu/ld-linux-*.so* /output/pkgs/lib64/ \
+    && (if test $(uname -m) = "x86_64"; then cp -r /usr/lib/*-linux-gnu/ld-linux-*.so* /output/pkgs/lib64/; fi) \
+    && (if test $(uname -m) = "aarch64"; then cp -r /usr/lib/*-linux-gnu/ld-linux-*.so* /output/pkgs/lib/; fi) \
```

Comment:
Can you build and upload an aarch64 sysroot to `https://github.com/redpanda-data/llvm-project/releases/download/llvmorg-19.1.7/sysroot-ubuntu-22.04-aarch64-2025-02-27.tar.zst`?

Reply:
Successfully built one, but I don't think I have permissions to upload to that release page.

Reply:
Updated the release assets here: https://github.com/redpanda-data/llvm-project/releases/tag/llvmorg-19.1.7

Can you update the build to use the one I just uploaded and verify it works on your arm machine?

Reply:
Yup!

Reply:
New sysroot is building without hitting that `libc.so.5 cannot find /lib/ld-linux-aarch64.so.1` error.

Updated the PR with the SHA256 and URL for the new aarch64 sysroot.